sdks/java/io/synthetic/build.gradle

```diff
@@ -29,11 +29,13 @@ dependencies {
   shadow library.java.jackson_annotations
   shadow library.java.jackson_databind
   shadow library.java.guava
+  shadow library.java.kafka_clients
```

Comment:
Why is this needed in `sdks/java/io/synthetic/build.gradle`? Shouldn't we add this in `sdks/java/testing/load-tests/build.gradle`?